Name in LogConfiguration.cpp

Hi,

In LogConfiguration.cpp is #include "versions.hpp" but file is named versions.hpp.in (from precice-1.3.0.zip).

Hi,
If I used cmake I have this error (with scons everything is OK):

```
C:/msys64/usr/local/precice-1.3.0/src/logging/LogConfiguration.cpp:16:10: fatal error: versions.hpp: No such file or directory
 #include "versions.hpp"
          ^~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/precice.dir/build.make:804: CMakeFiles/precice.dir/src/logging/LogConfiguration.cpp.obj] Błąd 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,8 +88,8 @@ endif()
 
 
 # Create versions.hpp and add binary dir to include dirs
-configure_file("${PROJECT_SOURCE_DIR}/src/versions.hpp.in" "${PROJECT_BINARY_DIR}/versions.hpp" @ONLY)
-include_directories(${PROJECT_BINARY_DIR})
+configure_file("${PROJECT_SOURCE_DIR}/src/versions.hpp.in" "${PROJECT_BINARY_DIR}/src/versions.hpp" @ONLY)
+include_directories("${PROJECT_BINARY_DIR}/src")
 
 add_subdirectory("src")
 include_directories("src")

